src/apps/hector-network/binance-smart-chain/hector-network.hec.token-fetcher.ts

```diff
@@ -0,0 +1,67 @@
+import { Inject } from '@nestjs/common';
+
+import { IAppToolkit, APP_TOOLKIT } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { ContractType } from '~position/contract.interface';
+import { PositionFetcher } from '~position/position-fetcher.interface';
+import { AppTokenPosition } from '~position/position.interface';
+import { Network } from '~types/network.interface';
+
+import { HectorNetworkContractFactory } from '../contracts';
+import { HECTOR_NETWORK_DEFINITION } from '../hector-network.definition';
+
+const appId = HECTOR_NETWORK_DEFINITION.id;
+const groupId = HECTOR_NETWORK_DEFINITION.groups.hec.id;
+const network = Network.BINANCE_SMART_CHAIN_MAINNET;
+
+@Register.TokenPositionFetcher({ appId, groupId, network })
+export class BinanceSmartChainHectorNetworkHecTokenFetcher implements PositionFetcher<AppTokenPosition> {
```

Comment:
Use the `AppTokenTemplatePositionFetcher` as done in `src/apps/hector-network/fantom/hector-network.s-hec-v2.token-fetcher.ts` (sorry, this isn't documented yet, but this is the new standard of writing app token fetchers)